app/views/layouts/application.html.erb

```diff
@@ -7,8 +7,12 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Prefer imports over CDN (not super important)

Reply:
Having a bit of trouble with this (seems like I'll have to do more than just an `@import "bootstrap"` in the application.js and remove the CDNs.

Will look into this later